orb/schema/validator.py

```diff
@@ -106,7 +106,7 @@ def validate(self, column, value):
 
         :return     <bool>
         """
-        if type(value) != bool and not bool(value):
+        if type(value) not in [bool, int, float] and not bool(value):
```

Comment:
Hi there! The preferred way to do this check is to use `isinstance`, passing a tuple of base types as the second argument:

```python
isinstance(value, (bool, int, float))
```
That will handle all possible subclasses.